quickstart/src/main/java/com/twilio/video/quickstart/util/CameraCapturerCompat.java

```diff
@@ -156,4 +158,26 @@ private boolean isPrivateImageFormatSupportedForCameraId(String cameraId) {
         isPrivateImageFormatSupported = streamMap.isOutputSupportedFor(ImageFormat.PRIVATE);
         return isPrivateImageFormatSupported;
     }
+
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+    private boolean isMonoChromeSupported(String cameraId) {
```

Comment:
The logic here is very similar to `isPrivateImageFormatSupportedForCameraId` ... Can you consolidate this logic into one method `isCameraIdSupported`

... Additionally can add these changes to `quickstartKotlin` and the multiparty quickstart.